proxies/venue-graphql-proxy/src/config/AppConfig.ts

```diff
@@ -6,6 +8,24 @@ class AppConfig {
   static get isHaukiEnabled() {
     return false;
   }
+
+  static get supportedLocales() {
+    return ['fi', 'en', 'sv'] as Locale[];
+  }
+
+  /** A prioritized list of fallback languages. */
+  static get fallbackLocales() {
+    return ['en', 'fi', 'sv'] as Locale[];
```

Comment:
This is different from the unified search's fallback languages order which is fi, en, sv. Just saying that we should use the same fallback languages order in both places. I don't know if we've had a real discussion anywhere about the fallback languages order anyway. Maybe we should.

Reply:
Hmm, the difference is only in the handling of missing Swedish translation:

If fallback language order is en/fi/sv (This PR currently):
- en -> fi/sv
- fi -> en/sv
- sv -> **en/fi**

If fallback language order is fi/en/sv (unified-search currently):
- en -> fi/sv
- fi -> en/sv
- sv -> **fi/en**

Reply:
I can switch unified search to en/fi/sv per [HH-110](https://helsinkisolutionoffice.atlassian.net/browse/HH-110):
> Logic for selecting the fallback language priority is needed, then prioritize order is English, Finnish, Swedish